pkg/e2etest/types.go

```diff
@@ -5,6 +5,7 @@ package e2etest
 
 import (
 	"context"
+	"github.com/els0r/goProbe/pkg/goDB/conditions/node"
```

Comment:
Please run `goimports` on all files upon saving, this breaks the standard ordering (internal packages first, then external ones).